html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
```

Comment:
I think this test is just throwing me off. This `<title>` indicates that we're only interested in the case where we do a history state change BEFORE the lazy loaded iframe is initially loaded, but then the test includes logic below to cater to the case where the "lazy loaded" iframe loads *before* the history state change takes place.

But we can easily fix this if we just move the iframe out of view, so that it is guaranteed to NOT load eagerly ever. Then the iframe is in a controlled state where we can perform the push state _always_ before the first load happens. Then we can load the iframe, and see what happens.

Reply:
Good point! We can use the `hidden` attribute for this.